addons-dev/aldryn-django/aldryn_django/templates/aldryn_django/configuration/nginx.conf

```diff
@@ -0,0 +1,61 @@
+user root root;
+
+events {
+  worker_connections 1024;
+}
+
+http {
+  include mime.types;
+  default_type application/octet-stream;
+
+  sendfile on;
+  keepalive_timeout 65;
+
+  proxy_connect_timeout 300s;
+  proxy_read_timeout 300s;
+  client_max_body_size 300m;
+
+  gzip on;
+  gzip_vary on;
+  gzip_comp_level 5;
+  gzip_min_length 10;
```

Comment:
AFAIK this is too low and will just add more overhead. 1K can be a better starting point.